src/util/psbt/serialize.rs

```diff
@@ -51,7 +52,7 @@ pub trait Deserialize: Sized {
 
 impl_psbt_de_serialize!(Transaction);
 impl_psbt_de_serialize!(TxOut);
-impl_psbt_de_serialize!(Vec<Vec<u8>>); // scriptWitness
+impl_psbt_de_serialize!(Witness); // scriptWitness
```

Comment:
I guess the comment can go away now?


```suggestion
impl_psbt_de_serialize!(Witness);
```